src/fidesops/schemas/analytics.py

```diff
@@ -0,0 +1,25 @@
+from typing import Optional, List, Dict
+
+from fidesops.schemas.base_class import BaseSchema
+
+
+class FideslogAnalyticsEvent(BaseSchema):
```

Comment:
If you check out [the fideslog `AnalyticsEvent` code](https://github.com/ethyca/fideslog/blob/main/fideslog/sdk/python/event.py), you'll see that it's already enforcing event validity for events created using the `AnalyticsEvent` constructor. This class shouldn't be needed unless you have plans to add new features on top of fideslog. If that's the case, maybe it's worth discussing if those features should be included in the SDK itself?

Reply:
`FideslogAnalyticsEvent` makes it easier to pass around necessary event-level data within fidesops with less duplication of code. Since `event_created_at `, `docker`, and `local_host` properties don't require event-specific info, we don't need to pass it in to `analytics_client.send()`. So, these props can be calculated only when we construct the `AnalyticsEvent`